metabase/metabase
# Backport various improvements from BigQuery driver branch :flushed:

Backport a bunch of random minor tweaks and fixes made in the `bigquery_driver_wip` branch to keep the scope of that PR more reasonable.
-  Make `is-prod?`/`is-test?`/`is-dev?` constants
-  Remove unused imports in `metabase/api/database.clj`
-  Remove unused `config-match` function
-  Add default implementation for `analyze-table`
-  Improve docstrings for `process-native` and `process-structured` by including expected structure of their responses
-  Fix typo in comment in `metabase/driver/druid.clj`
-  Fix excess whitespace in `metabase/driver/druid.clj`
-  Fix arguments in wrong order in protocol declaration of `unix-timestamp->timestamp`
-  Fix misaligned comments in `src/metabase/driver/generic_sql/native.clj`
-  Simplify code for preparing objects in SQL query processor 
-  Simplify korma form logged by SQL query processor (i.e. `korma.core$do_query` instead of `#object[korma.core$do_query korma.core$do_query @ 0x0f431267]`)
-  Increase pprint width for logging queries, expanded queries, and korma forms, making logging more concise
-  Make `events-init` functions private
-  Addidtional docstrings form most of the functions in `metabase.util.korma-extensions`
